packages/grid/x-data-grid/src/components/ErrorOverlay.tsx

```diff
@@ -14,9 +16,10 @@ export const ErrorOverlay = React.forwardRef<HTMLDivElement, ErrorOverlayProps>(
     const { message, hasError, errorInfo, ...other } = props;
     const apiRef = useGridApiContext();
     const defaultLabel = apiRef.current.getLocaleText('errorOverlayDefaultLabel');
+    const rowHeight = useGridSelector(apiRef, gridDensityRowHeightSelector);
 
     return (
-      <GridOverlay ref={ref} {...other}>
+      <GridOverlay ref={ref} sx={{ width: '100%', minHeight: 2 * rowHeight }} {...other}>
```

Comment:
Ok to keep `minHeight` even with `autoHeight=false` because the grid has `overflow: hidden`. If it didn't hide what overflows this could be a problem.

If you want to go further, the `sx` prop converts values between [0, 1] to a percentage. See  https://mui.com/system/the-sx-prop/#sizing

```suggestion
      <GridOverlay ref={ref} sx={{ width: 1, minHeight: 2 * rowHeight }} {...other}>
```

Reply:
> Ok to keep minHeight even with autoHeight=false because the grid has overflow: hidden. If it didn't hide what overflows this could be a problem

Yeah, I was thinking about applying it conditionally, but as you said - it won't cause any harm anyway.

> sx prop converts values between [0, 1] to a percentage

Thanks, I didn't know about that.
In this particular case I prefer explicit `100%` though